src/frontends/tensorflow_lite/include/openvino/frontend/tensorflow_lite/sparsity_info.hpp

```diff
@@ -30,12 +31,16 @@ class SparsityInfo : public ov::RuntimeAttribute {
                           const std::vector<int32_t>& traversal_order,
                           const std::vector<int32_t>& block_map,
                           const std::vector<uint16_t>& dim_format,
-                          const std::vector<SparsityDataDesc>& data_desc)
+                          const std::vector<SparsityDataDesc>& data_desc,
+                          const ov::element::Type target_type,
+                          const uint8_t* values)
         : m_shape(shape),
           m_traversal_order(traversal_order),
           m_block_map(block_map),
           m_dim_format(dim_format),
-          m_data_desc(data_desc) {
+          m_data_desc(data_desc),
```

Comment:
can you give description to `SparsityInfo` class? From your perspective, you should know for which cases it is needed.
It will be very helpful.